markdown/bitburner.ns.scan.md

```diff
@@ -9,25 +9,24 @@ Get the list servers connected to a server.
 <b>Signature:</b>
 
 ```typescript
-scan(host: string, hostnames?: boolean): string[];
+scan(host?: string): string[];
```

Comment:
Like i said before, hydroflame and many other contributors told me to remove it because the scan function wasn't using the ip parameter and the logic wasn't there to return ips so it was redundant and confusing for people using that function because the intellisense said you could set it to false to return ips but when called or used it didn't work as instructed. So we all decided in the discord to remove it and the docs associated with it. Also, a lot of the changes above are automatic i only manually changed the scan function and its docs the rest automatically changed.

Reply:
Also
![9b0228998ff803e8eda73fa8675dcfd7](https://user-images.githubusercontent.com/13303466/146408841-a6980ed2-d636-4fb0-8b20-20dae6133060.png)

Reply:
ye.... a review was a bit too much 😆